src/elbo.jl

```diff
@@ -68,9 +80,14 @@ function _elbo(
     q_f = marginals(post(fx.x))
     variational_exp = expected_loglik(quadrature, y, q_f, lik)
 
-    kl_term = KL(sva.q, sva.fz)
-
     n_batch = length(y)
     scale = num_data / n_batch
-    return sum(variational_exp) * scale - kl_term
+    return sum(variational_exp) * scale - kl_term(sva, post)
+end
+
+kl_term(sva::SparseVariationalApproximation, post) = KL(sva.q, sva.fz)
+
+function kl_term(sva::WhitenedSparseVariationalApproximation, post)
```

Comment:
would be great if we could have an explicit test for this (should be equal to `KL` when un-whitened)